blazor/datagrid/excel-like-filter.md

```diff
@@ -48,15 +48,13 @@ You can enable Excel like filter by defining [Type](https://help.syncfusion.com/
 
 ```
 
-The following screenshot represents Excel filter
 ![Blazor DataGrid with Excel Filter](./images/blazor-datagrid-excel-filter.png)
 
-The following screenshot represents Custom filter in Excel filter
```

Comment:
they added this details to differentiate the images.

Reply:
Okay, I'll add it again.